crates/hyperdrive-math/src/short/max.rs

```diff
@@ -48,7 +48,7 @@ impl State {
     /// on the realized price that the short will pay. This is used to help the
```

Comment:
A few lines up change to

Calculates the max short that can be opened with the given budget.

Reply:
good catch!